pkg/internal/codegen/parse/crd.go

```diff
@@ -577,6 +581,20 @@ func getValidation(comment string, props *v1beta1.JSONSchemaProps) {
 		}
 	case "Format":
 		props.Format = parts[1]
+	case "Title":
+		title := strings.TrimSpace(strings.Join(parts[1:], " "))
+		if len(title) > 2 {
+			// Remove beginning and ending quote from non-empty strings.
+			if title[0] == '"' {
```

Comment:
this accepts `Title=foo"`, which doesn't seem to be valid.  Also, `Title=f` should probably be valid.  I think we may need to hold on here a bit and define more format parsing semantics for our markers (I've got some stuff in the works that I'll post soon).

Reply:
We could simply require string fields be wrapped in `""` and anything inside will be interpreted literally.

Reply:
I've got parsing logic written.  We should be consistent across our different annotations.

Reply:
Has it been merged?

Reply:
not yet, it's part of a broader overhaul being tracked on the crdgenerator feature branch